extension/lib/shopgate/CartExtensionPipeline.js

```diff
@@ -86,6 +86,14 @@ class ShopgateCartExtensionPipeline {
     return this._bigCommerceCartRepository.getCheckoutUrl()
   }
 
+  /**
+   * @param {Array} cartItemIds
+   * @return {Promise.<void>}
```

Comment:
We're usually using syntax without dot: `Promise<void>`.

Reply:
done